flake.nix

```
@@ -32,6 +34,17 @@
           '';
         };
 
+        emanote = {
+          baseUrl = "https://github.com/srid/emanote";
```

```suggestion
          baseUrl = "https://github.com/srid/emanote/blob/master";
```

Some code parses this. It should take the same information from specific individual options instead. I'll improve that later. For now this should make it work.